src/InputPassword/inputPassword.style.js

```diff
@@ -1,18 +1,20 @@
 import styled from "styled-components";
 
 const InputPasswordStyles = styled.div`
-  .icon-position-right {
-    font-size: 30px;
-    cursor: pointer;
+  position: relative;
```

Comment:
why not flex based?

Reply:
because the button that we add on the right side of the Input needs to be there positioned `absolute`. 
Wanna see some screenshots?

Reply:
that will be cool

Reply:
![image](https://user-images.githubusercontent.com/4181674/48892247-efe55500-ee3d-11e8-851d-364def71d214.png)
![image](https://user-images.githubusercontent.com/4181674/48892256-f5429f80-ee3d-11e8-9e17-600ba370b656.png)

this icon on the right side is being positioned there by the class `.button-position-right`, and for absolute to work nicely I made the parent component `position: relative;`